Sources/BraintreePayPalMessaging/BTPayPalMessagingLogoType.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import PayPalMessages
+
+/// Logo type option for a PayPal Message
+/// - Note: This module is in beta. It's public API may change or be removed in future releases.
+public enum BTPayPalMessagingLogoType {
+
+    /// PayPal logo positioned inline within the message
```

Comment:
we'll eventually want to include a link to docs showing what these different logos look like

Reply:
Agree - we can have the Messages team include that when we get to writing docs for this module.